### PR TITLE
Consolidate common code808

### PR DESCRIPTION
**Pull request by jamiesjc**
Combined common code to configure job properties in one method.
Changes based on comments in Pull request #807.

### CODE REVIEW

File: azkaban-common/src/test/java/azkaban/jobExecutor/AllJobExecutorTests.java [hunk 1]
```diff
@@ -19,10 +19,24 @@
...
 import org.junit.runners.Suite.SuiteClasses;
+import azkaban.flow.CommonJobProperties;
+import azkaban.utils.Props;
 
 @RunWith(Suite.class)
 @SuiteClasses({ JavaProcessJobTest.class, ProcessJobTest.class,
     PythonJobTest.class })
 public class AllJobExecutorTests {
 
+  public void setUpCommonProps(Props props){
```
Comment (fsi206914) [inline azkaban-common/src/test/java/azkaban/jobExecutor/AllJobExecutorTests.java]: Static Methods might be more appropriate here.

File: azkaban-common/src/test/java/azkaban/jobExecutor/JavaProcessJobTest.java [hunk 2]
```diff
@@ -45,6 +45,7 @@
   private JavaProcessJob job = null;
   private Props props = null;
   private Logger log = Logger.getLogger(JavaProcessJob.class);
+  private AllJobExecutorTests jobExecutorTests = null;
```
Comment (HappyRay) [inline azkaban-common/src/test/java/azkaban/jobExecutor/JavaProcessJobTest.java]: What's the benefit of having jobExecutorTests as a member variable? 
What do you think about making the method static? Afterall, it is a stateless method.

Comment (HappyRay) [inline azkaban-common/src/test/java/azkaban/jobExecutor/JavaProcessJobTest.java]: Never mind. I saw you changed it already in a later commit.

**Comment by HappyRay** [comment]
LGTM

**Comment by HappyRay** [comment]
Reminder, please use the github feature to clean up the commit message before you merge.

**Pull request merged**